services/php-apache-node/Dockerfile

```diff
@@ -12,14 +12,8 @@ RUN git clone https://github.com/tj/n.git \
     && apt-get update \
     && apt-get --quiet install -y --no-install-recommends \
       libzip-dev \
-      parallel \
-      python3-pip \
       redis-tools \
       software-properties-common \
-    && python3 -m pip install --upgrade pip \
-    && pip3 install --upgrade setuptools \
-    && pip3 install wheel \
-    && pip3 install ansible awscli pyyaml \
```

Comment:
As far as I know, these tools are no longer used, we use Usher/robo instead of Ansible, the PHP AWS library instead of the CLI and I don't remember why we needed pyyaml.